src/components/Card/Card.component.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Grid } from '@material-ui/core';
+
+const StyledCard = styled.div`
```

Comment:
I'd like to suggest you to move your styled components outside the main component. As these can get quite big due to CSS, it is better to have them in a separated file, it is usually called `styled.js`.